app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -26,5 +28,21 @@ def self.extract_disability_symbol_list(form526_submission)
       form_disabilities = form526_submission.form.dig('form526', 'form526', 'disabilities')
       form_disabilities.map { |form_disability| DISABILITIES_BY_CODE[form_disability['diagnosticCode']] }
     end
+
+    # @return [Hash] for the first RRD-supported disability in the form526_submission
+    def self.first_disability(form526_submission)
+      extracted_disability_symbols = extract_disability_symbol_list(form526_submission)
+      return if extracted_disability_symbols.empty?
+
+      disability_symbol = extracted_disability_symbols.first
+      DISABILITIES[disability_symbol]
+    end
+
+    def self.processor_class(form526_submission)
+      disability_struct = first_disability(form526_submission)
+      return unless disability_struct
+
+      disability_struct[:processor_class]&.constantize
+    end
```

Comment:
Neat. I vaguely recall this coming up but what happens in the case of multiple disabilities?

Reply:
Great question! The short answer is we may have to update `Constants` when we have a design to handle multiple disabilities.
In the meantime, rather than thinking "given a _disability_, what is the associated sidekiq job and processor to use?" (which is what `Constants` provides now), we should start thinking of the mapping as "given a _claim_, what is the associated sidekiq job and processor to use?". The `ProcessorSelector` (which I renamed to `SidekiqJobSelector`) and the chosen sidekiq job would provide the answer to that question.